src/main/fc/rc_modes.c

```diff
@@ -72,18 +71,36 @@ bool isRangeActive(uint8_t auxChannelIndex, const channelRange_t *range) {
 
 void updateActivatedModes(void)
 {
-    boxBitmask_t newMask;
+    int index;
```

Comment:
By convention, variables are defined on first use in Betaflight (today's compilers are good enough to optimise them). Please change this back.

Reply:
I pulled it out because it is used in separate for-loops but will change it to define in both.

Reply:
From a 'I have to optimise' point of view this is good - but you don't have to, gcc will do this for you.

From the point of view that at the first glance, having the variable pulled out makes it look to a casual observer like state is meant to be transferred between the loops, having the variables defined inside the loops makes it easier to read.